PWGHF/TableProducer/treeCreatorToXiPi.cxx

```diff
@@ -35,20 +33,10 @@ namespace o2::aod
 namespace full
 {
 // collision info
-DECLARE_SOA_INDEX_COLUMN(Collision, collision);
-DECLARE_SOA_INDEX_COLUMN(McCollision, mcCollision);
 DECLARE_SOA_COLUMN(IsEventSel8, isEventSel8, bool);
 DECLARE_SOA_COLUMN(IsEventSelZ, isEventSelZ, bool);
-DECLARE_SOA_COLUMN(RunNumber, runNumber, int);
-DECLARE_SOA_COLUMN(IsEventReject, isEventReject, int);
-DECLARE_SOA_COLUMN(CentFT0A, centFT0A, float);
-DECLARE_SOA_COLUMN(CentFT0C, centFT0C, float);
-DECLARE_SOA_COLUMN(CentFT0M, centFT0M, float);
-DECLARE_SOA_COLUMN(CentFV0A, centFV0A, float);
-DECLARE_SOA_COLUMN(CentFDDM, centFDDM, float);
-DECLARE_SOA_COLUMN(MultZeqNTracksPV, multZeqNTracksPV, float);
+DECLARE_SOA_COLUMN(Centrality, centrality, float);
```

Comment:
If you just want a specific centrality estimator, please specify this info in the name of the column.